kolibri/plugins/coach/assets/src/state/actions/exam.js

```diff
@@ -143,50 +123,28 @@ export function setExamsModal(store, modalName) {
   store.dispatch('SET_EXAMS_MODAL', modalName);
 }
 
-export function activateExam(store, examId) {
+function updateExamStatus(store, { examId, isActive }) {
   return ExamResource.getModel(examId)
-    .save({ active: true })
+    .save({ active: isActive })
     .then(
       () => {
-        const exams = store.state.pageState.exams;
-        const examIndex = exams.findIndex(exam => exam.id === examId);
-        exams[examIndex].active = true;
-
-        store.dispatch('SET_EXAMS', exams);
+        store.dispatch('SET_EXAM_STATUS', { examId, isActive });
         setExamsModal(store, false);
-
         createSnackbar(store, {
-          text: createTranslator('examActivateSnackbar', {
-            examIsNowActive: 'Exam is now active',
-          }).$tr('examIsNowActive'),
+          text: snackbarTranslator.$tr(isActive ? 'examIsNowActive' : 'examIsNowInactive'),
```

Comment:
will this work as expected? For some reason I thought we couldn't use expressions inside of `$tr` functions (cc @rtibbles)